glyphs-reader/src/font.rs

```diff
@@ -325,6 +325,24 @@ fn custom_param<'a>(
     None
 }
 
+fn glyphs_v2_field_name_and_default(nth_axis: usize) -> Result<(&'static str, f64), Error> {
+    // Per https://github.com/googlefonts/fontmake-rs/pull/42#pullrequestreview-1211619812
+    // the field to use is based on the order in axes NOT the tag.
+    // That is, whatever the first axis is, it's value is in the weightValue field. Long sigh.
+    // Defaults per https://github.com/googlefonts/fontmake-rs/pull/42#discussion_r1044415236.
+    Ok(match nth_axis {
+        0 => ("weightValue", 400_f64),
```

Comment:
the weightValue default is actually 100, in design coordinates, but I see you return 400 in user-space coordinates, deliberately I suppose...

Reply:
Fixed